rtgui/batchqueue.cc

```diff
@@ -94,6 +95,21 @@ namespace   // local helper functions
             }
         }
     }
+
+    // Look in templateText at index ix for quoted string containing a time format string, and
+    // use that string to format dateTime. Append the formatted time to path.
+    void appendFormattedTime(Glib::ustring& path, unsigned int& ix, const Glib::ustring& templateText, const Glib::DateTime& dateTime)
+    {
+        constexpr unsigned int quoteMark = (unsigned int)'"';
```

Comment:
This looks wrong. The C-cast is wrong on its own, but `unsigned int` is even more wrong.

Reply:
Changing to `gunichar('"')`